test/create-snowpack-app/__snapshots__/create-snowpack-app.test.js.snap

```diff
@@ -7381,7 +7381,7 @@ exports[`create-snowpack-app app-template-minimal > build: package.json 1`] = `
     \\"test\\": \\"echo \\\\\\"This template does not include a test runner by default.\\\\\\" && exit 1\\"
   },
   \\"devDependencies\\": {
-    \\"snowpack\\": \\"^2.18.1\\"
```

Comment:
whoops forgot to regenerate these after publishing a couple of minutes ago, just updated snapshots on main to fix this.